translations/translations.json

```
@@ -4210,6 +4211,7 @@
     },
     "en": {
       "address": "Address",
+      "meetingUrl": "Online-Meeting",
```

🔧 I think in English you write it without a hyphen:
```suggestion
      "meetingUrl": "Online meeting",
```